spec/models/user_spec.rb

```diff
@@ -32,5 +36,31 @@
         it { is_expected.to_not allow_value(phone).for(:phone) }
       end
     end
+
+    context 'user' do
+      context 'on create' do
```

Comment:
`describe`, `context` and `it` together should forms valid eng sentence
please read: http://www.betterspecs.org/